services/src/unit-test/java/uk/gov/moj/sdt/services/messaging/MessageWriterTest.java

```diff
@@ -70,6 +67,7 @@ void testQueueMessageWithEmptyTargetApp() {
         // Send the message.
         try {
             messageWriter.queueMessage(sdtMessage, null, false);
+            verify(mockJmsTemplate).convertAndSend("", sdtMessage);
```

Comment:
I don't think this check provides any benefit here.

An IllegalArgumentException should be thrown when queueMessage() is called and convertAndSend() never invoked.  If the exception isn't thrown then something has gone wrong and the test should be failed.  In this situation it's not really worth checking whether convertAndSend() has been called or not, all that needs to be done is to fail the test which the following fail() does.

You could, however, improve the test by checking that convertAndSend() was not called using `verify(mockJmsTemplate, never()).convertAndSend(anyString(), any(ISdtMessage.class));`